test/unit/parsers/builder.test.ts

```diff
@@ -6,7 +8,7 @@ import {
 
 describe("parsers/builder.ts", () => {
   describe("::buildSafeParser", () => {
-    let parseFn;
+    let parseFn: jest.MockedFunction<ParseFn<unknown>>;
```

Comment:
This can be simply:

```suggestion
    let parseFn: ReturnType<typeof jest.fn>;
```

in which case the added import can be removed as well.